woocommerce/woocommerce-ios
# [WOO POS] create and sync order, collect payment

#12763 #13021 #12983


## Description


- Updated PointOfSaleDashboardViewModel to include logic for creating and syncing of the order
- created order will be used for collecting the payment
- processing the payment
- UI will be finalized in one of the next PRs alogside with better order cleanup for next transaction and payment processing

## Testing information


- Enter POS experience
- Add few products in the cart
- Tap "Checkout"
- Check that the totals and payments screen is shown with totals amounts being calculated and shown
- Use the card reader to collect the payment
- Check that the payment is successful and "New transaction" button is shown

## Screenshots


![order payment](https://github.com/woocommerce/woocommerce-ios/assets/6242034/b021844e-83f3-40d9-81e9-a0f3c1dbd8ce)

---
